src/main/java/net/ltxprogrammer/changed/ChangedConfig.java

```diff
@@ -41,12 +39,20 @@ private interface AdditionalData {
     public static class Common {
         public final ForgeConfigSpec.ConfigValue<String> githubDomain;
         public final ForgeConfigSpec.ConfigValue<Boolean> displayPatronage;
+        public final ForgeConfigSpec.ConfigValue<Boolean> enableTransfurringOrigins;
```

Comment:
This configuration should actually be set in the `SERVER` configuration, so that it syncs to all connected clients.